code/controllers/subsystems/holomap.dm

```diff
@@ -41,15 +39,12 @@ SUBSYSTEM_DEF(minimap)
 			single.Blend(A.holomap_color, ICON_MULTIPLY)
 		combinedareas.Blend(single, ICON_OVERLAY)
 
-	data.holomap_areas_combined = combinedareas
-
 	var/icon/map_base = icon(data.holomap_base)
 
 	// Generate the full sized map by blending the base and areas onto the backdrop
 	var/icon/big_map = icon(HOLOMAP_ICON, "stationmap")
```

Comment:
This variable is now unused also, which means we're wasting two 480x480 blend ops.